### ISSUE TITLE
508-defect-3 [FOCUS MANAGEMENT]: Focus outline SHOULD be consistent - individual nav links

### ISSUE THREAD

**Issue by jenstrickland** [department-of-veterans-affairs/va.gov-team#8975]
# [508-defect-3](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-3)

**Feedback framework**

- **❗️ Must** for if the feedback must be applied 
- **⚠️Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Description

For viewports 768px wide and less, the focus outline for the side nav links **should** be consistent with focus styling across VA.gov. It is currently cut off on the left and right sides.

## Point of Contact

**VFS Point of Contact:** Jennifer

## Acceptance Criteria

As a keyboard user, I want to know with certainty where the focused area is, and for it to be a consistent cue across the site.

## Environment

* Operating System: all
* Browser: all
* Server destination: https://staging.va.gov/pittsburgh-health-care/

**Viewports less than 768px wide**

## Steps to Recreate

1. Enter https://staging.va.gov/pittsburgh-health-care/ in browser
2. Resize browser to less than 768 pixels wide
3. Select the "In this section" link to open the secondary nav menu
4. Select an item in the nav options
5. Verify the focus outline is cut off on the left and right

### Current global focus outline
```diff
.usa-focus, [contentEditable=true]:focus, [href]:focus, [tabindex]:focus, button:not([disabled]):focus, iframe:focus, input:not([disabled]):focus, select:not([disabled]):focus, textarea:not([disabled]):focus {
    outline: 2px solid #f9c642;
    outline-offset: 2px;
}
```

### Recommended 
Add a focus style for this item

```diff
#va-sidenav-ul-container [href]:focus {
    outline-offset: -2px;
}
```

## WCAG or Vendor Guidance (optional)

* [W3C - WCAG 2.0 Consistent Identification](https://www.w3.org/TR/UNDERSTANDING-WCAG20/consistent-behavior-consistent-functionality.html)

## Screenshots or Trace Logs

![Screen Shot 2020-05-11 at 5 08 55 PM (2)](https://user-images.githubusercontent.com/57469/81612772-5bf9a700-93ab-11ea-88f3-7b32583eb4fe.png)

**Comment by mmiddaugh** [comment]
This issue was called out during staging review so should be addressed, even though it is coded defect-3

**Comment by mmiddaugh** [comment]
@jenstrickland this is ready for your review

**Comment by jenstrickland** [comment]
@mmiddaugh I'm still seeing the same styling. Is this intended? The consistent focus styling on VA.gov is that it appear around all four sides.


![Screen Shot 2020-06-24 at 1 30 53 PM](https://user-images.githubusercontent.com/57469/85604076-f55cdf80-b61e-11ea-8e5c-6fd4887f2ee3.png)

**Comment by brangi** [comment]
@mmiddaugh I have a fix for this - https://github.com/department-of-veterans-affairs/vets-website/pull/15559 however this css value is not supported on ie11 https://caniuse.com/?search=outline-offset . I can merge the current fix but we should open a new issue for ie11 support. I feel that this is more of a VSP design system effort since we use `outline-offset` all over the site

**Comment by mmiddaugh** [comment]
Created #18269 for IE-11 persistence and posted in the design channel for visibility
